usr/share/rear/output/default/940_grub2_rescue.sh

```diff
@@ -158,17 +160,8 @@ if is_true $USING_UEFI_BOOTLOADER ; then
         echo "}"
     ) > $grub_config_dir/rear.cfg
 
-    # Tell rear.efi which configuration file to load
-    (   echo "search --no-floppy --fs-uuid --set=root $grub_boot_uuid"
-        echo ""
-        echo "set btrfs_relative_path=y"
-        echo "set prefix=(\$root)${grub_boot_dir}/grub${grub_num}"
-        echo ""
-        echo "configfile (\$root)${grub_boot_dir}/grub${grub_num}/rear.cfg"
-    ) > $grub_config_dir/rear_embed.cfg
```

Comment:
Why did you actually removed this whole block ?
Are you 100% sure it won't be missing by something or someone?
IMHO the `prefix` might be useful for some distros ...

V.

Reply:
I returned `set btrfs_relative_path=y`, as it was indeed needed.
We have `search --no-floppy --fs-uuid --set=root $grub_boot_uuid` already, see above.
`configfile` is not needed, because with grub-mkstandalone, we are not using two config files, just one, so a command to load the other one is not needed.
Concerning prefix, see https://www.gnu.org/software/grub/manual/grub/grub.html#prefix . This is a directory from where Grub2 loads its modules. grub-mkstandalone sets it to a correct value: to a memdisk embedded in the standalone image. This ensures that all the modules will be always accessible. I don't think we should change it; this would only risk breaking autoloading of modules.
Perhaps you are confusing with `root`; this is useful, but is set by the `search --no-floppy --fs-uuid --set=root $grub_boot_uuid` command (no change here).

Reply:
I know `prefix` does and I'm not confusing it with `root`, but through couple of past years, I saw way too many different strange behavior and non-functional boot images that's why I don't like to see code being removed just because current version should work in certain way. 
In upcoming days I'll test your code on couple of other distros and see how it behaves.

Thanks for your work for now.

V.

Reply:
@gozora it would be great to have the code tested on more distros, thanks! Note that it is expected to break on Ubuntu 12.04, see https://github.com/rear/rear/pull/2199#issuecomment-561602385 . Other distros should be fine.

Reply:
By the way, I would be interested if it fixes your btrfs-on-lvm case without the need of setting GRUB2_MODULES manually...

Reply:
Yes good point, I'll test that. 
But don't expect results too soon. My current higher activity in ReaR community is more an happy accident and in upcoming weeks I'll not have more that 2-3 hours a week :-( ...